docker/transformers-pytorch-amd-cpu-zentorch/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 The HuggingFace Team All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG UBUNTU_VERSION=20.04
+
+FROM ubuntu:${UBUNTU_VERSION}
+
+ARG TORCH_VERSION=2.2.1
+
+# Install python and g++ compiler
+ENV DEBIAN_FRONTEND noninteractive
+ENV PATH="/home/user/.local/bin:${PATH}"
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git \
+    ffmpeg \
+    python3.8 \
+    python3-pip \
+    python3.8-dev \
+    build-essential \
+    libjemalloc-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1 && \
```

Comment:
Maybe just to keep in mind that python 3.8 soon reaches end-of-life

Reply:
the plugin we have right now is compiled with Python3.8
https://huggingface.slack.com/archives/C06GVGQA0JG/p1710936009492349